### DISCUSSION TITLE
local dagster dev run give error

### DISCUSSION THREAD

**Discussion by aura-edwu** [dagster-io/dagster#24918] [Q&A]
Hi i was developing something on my local env. when i run dagster dev this morning everything works fine but now i'm getting this error
ModuleNotFoundError: No module named 'dagster._core.storage.batch_asset_record_loader'

**Comment by garethbrickman** [comment]
Were you possibly using an older version of Dagster in your code and then installed a newer version in your local environment? Or changed your local environment so that it referenced a different version? 

Either way, as a first approach to resolving it you could create a fresh virtual environment and install your Dagster project dependencies again. If your code is based from an older version of Dagster you should pin to that version in your `setup.py`/`requirements.txt`
